templates/meson.build

```diff
@@ -1,5 +1,6 @@
 install_subdir('gtk', install_dir: templates_dir)
 install_subdir('lib', install_dir: templates_dir)
 install_subdir('new', install_dir: templates_dir)
+install_subdir('elementary-OS-app', install_dir: templates_dir)
```

Comment:
`elementary-OS-app` as a template name is unnecessarily long. Use `eos` instead.

Reply:
eos is confusing, i recommend 'elementary', or 'elementary-app'

Reply:
What about `eOS`?

Reply:
I thought they dropped the OS, just elementary now

Reply:
The other thing is that we list the template names along with a short description, so I don't think `eos` would be confusing in that context.

```
% valdo                                                                                                                                                             ✹ ✭
Available templates:
--------------------
lib               - a bare library with minimal dependencies
elementary-OS-app - a starter elementary OS app
gtk               - a starter GTK3 app
swifty-gtk4       - an opinionated Gtk4 application template
new               - a bare app, with minimal dependencies
```

Reply:
@benwaffle 

![Screenshot from 2021-07-29 14-17-07](https://user-images.githubusercontent.com/1341033/127544746-c4537a3c-caed-48f0-86ec-25b1cfdf9cff.png)

Reply:
swifty makes me think it is written in Swift

Reply:
![Screenshot from 2021-07-30 05-52-30](https://user-images.githubusercontent.com/68198116/127581600-634b58ec-1eba-4ede-a793-597e1a157c1f.png)

This is why I used elementary OS instead of eOS or anything like that. eOS can be confused with other distros like endeavour OS and others

Reply:
The point of the template names is to abbreviate, though. I agree with that statement when referring to elementary OS explicitly, but Valdo templates are supposed to be almost like (short) keywords, since we don't want the user to have to type a lot.

Reply:
So should we use eOS?

Reply:
Use `eos`, which doesn't require the user to hit the shift key.